packages/storybook/stories/va-back-to-top.stories.jsx

```diff
@@ -9,24 +9,28 @@ export default {
 
 const Template = () => {
   return (
-    <main className="vads-l-col--12 medium-screen:vads-l-col--8">
-      <h1>The top</h1>
-      {[...Array(10)].map(i => (
-        <div
-          key={i}
-          style={{
-            height: '300px',
-            background: 'whitesmoke',
-            margin: '2em 0',
-            display: 'flex',
-            alignItems: 'center',
-            justifyContent: 'center',
-          }}
-        >
-          Scroll down
+    <>
+      <div className="usa-grid usa-grid-full">
+        <div className="usa-width-three-fourths">
+          <h1>The top</h1>
+          {[...Array(10)].map(i => (
+            <div
+              key={i}
+              style={{
+                height: '300px',
+                background: 'whitesmoke',
+                margin: '2em 0',
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'center',
+              }}
+            >
+              Scroll down
+            </div>
+          ))}
+          <va-back-to-top />
```

Comment:
We moved the footer out of the main content container which fixes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/583.

This also makes `<va-back-to-top>` the last child, so that we can make sure the styling change applies.